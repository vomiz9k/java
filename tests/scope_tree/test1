class Main {
    public static void main() {
    }
}


class jija {
    int q;
    public int joja(int a) {
        int b;
        {
            int c;
            int d;
        }
        {
            int h;
        }
        int f;
        System.out.println(f);
    }
}