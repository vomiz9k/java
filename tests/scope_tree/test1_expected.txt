Main jija
	q joja
		a 
			b f
				c d
				h 
